### PR TITLE
update README

### PR DESCRIPTION
**Pull request by tuxcanfly**
yep.

### CODE REVIEW

File: docs/Beginner's-Guide.md [hunk 1]
```diff
@@ -11,36 +11,30 @@ Bcoin is an _alternative_ implementation of the bitcoin protocol, written in nod
...
 
-(\*): Note that bcoin works best with unix-like OSes, and has not yet been thoroughly tested on windows.
+(\*): Note that bcash works best with unix-like OSes, and has not yet been thoroughly tested on windows.
 
 (\*\*): The BSDs and Solaris have also not been tested yet, but should work in theory.
 
 ## Build & Install
 
-Bcoin is meant to be installed via npm, but for the security conscious, it may be better to clone from github. All tagged commits for release should be signed by @chjj's [PGP key][keybase] (`B4B1F62DBAC084E333F3A04A8962AB9DE6666BBD`). Signed copies of node.js are available from [nodejs.org][node], or from your respective OS's package repositories.
+Bcoin is meant to be installed via npm, but for the security conscious, it may be better to clone from github.
```
Comment (bucko13) [inline docs/Beginner's-Guide.md]: bcash

File: docs/Beginner's-Guide.md [hunk 2]
```diff
@@ -11,36 +11,30 @@ Bcoin is an _alternative_ implementation of the bitcoin protocol, written in nod
...
-$ git tag -v v1.0.0-alpha # verify signature
-$ git checkout v1.0.0-alpha
+$ git clone git://github.com/bcoin-org/bcash.git
+$ cd bcash
 $ npm install -g --production
 ```
 ### Installing via Docker
 
-Check [bcoin-docker](https://github.com/bcoin-org/bcoin-docker)
+Check [bcash-docker](https://github.com/bcoin-org/bcash-docker)
```
Comment (bucko13) [inline docs/Beginner's-Guide.md]: No support yet right?

File: docs/Beginner's-Guide.md [hunk 3]
```diff
@@ -136,27 +130,27 @@ Your hidden service must first be configured with `tor`. Once you have the `.oni
...
 
 ## Disabling Listening
 
-To avoid accepting connections on the P2P network altogether, `--listen=false` can be passed to bcoin.
+To avoid accepting connections on the P2P network altogether, `--listen=false` can be passed to bcash.
 
 ### Selfish Mode
 
-Bcoin also supports a "selfish" mode. In this mode, bcoin still has full blockchain and mempool validation, but network services are disabled: it will not relay transactions or serve blocks to anyone.
+Bcoin also supports a "selfish" mode. In this mode, bcash still has full blockchain and mempool validation, but network services are disabled: it will not relay transactions or serve blocks to anyone.
```
Comment (bucko13) [inline docs/Beginner's-Guide.md]: `Bcash also supports`

File: docs/CLI.md [hunk 4]
```diff
@@ -5,68 +5,68 @@ Bcoin ships with bcoin-cli as its default HTTP client for command line access.
...
+$ bcash rpc getpeerinfo
+$ bcash rpc getbalance
+$ bcash rpc listtransactions
+$ bcash rpc sendtoaddress [address] 0.01
 ```
 
 ## Commands
 
-bcoin-cli commands are split into 3 categories: cli, rpc, and wallet.
+bcash-cli commands are split into 3 categories: cli, rpc, and wallet.
```
Comment (bucko13) [inline docs/CLI.md]: this is still bcoin-cli right?

**Comment by tuxcanfly** [comment]
Merged.

**Pull request closed without merging**